android/app/src/main/java/com/microsoft/codepush/common/datacontracts/CodePushLocalPackage.java

```diff
@@ -16,6 +16,12 @@
     @SerializedName("isPending")
     private boolean isPending;
 
+    /**
+     * Indicates the path to the application entry point.
+     */
+    @SerializedName("appEntryPoint")
+    private String appEntryPoint;
```

Comment:
Please be careful with properties names changes of local manifests and etc. I mean in terms of backward compatibility.